docs/source/intro/configure.rst

```diff
@@ -0,0 +1,155 @@
+
+Each app reference in the :setting:`INSTALLED_APPS` setting must point to the
+corresponding ``AppConfig``. For example, instead of ``'django.contrib.admin'``,
+the template uses ``'<project_name>.apps.MongoAdminConfig'``.
+
+Configuring migrations
+======================
+
+Because all models must use
+:class:`~django_mongodb_backend.fields.ObjectIdAutoField`, each third-party
+and contrib app you use needs to have its own migrations specific to MongoDB.
+
+For example, ``settings.py`` in the project template specifies::
+
+    MIGRATION_MODULES = {
+        "admin": "mongo_migrations.admin",
+        "auth": "mongo_migrations.auth",
+        "contenttypes": "mongo_migrations.contenttypes",
+    }
+
+The project template includes these migrations, but you can generate them if
+you're setting things up manually or if you need to create migrations for
+third-party apps. For example:
+
+.. code-block:: bash
+
+    $ python manage.py makemigrations admin auth contenttypes
+    Migrations for 'admin':
+      mongo_migrations/admin/0001_initial.py
+        - Create model LogEntry
+    ...
+
+Creating Django applications
+============================
+
+Whenever you run ``python manage.py startapp``, you must remove the line::
+
+    default_auto_field = 'django.db.models.BigAutoField'
+
+from the new application's ``apps.py`` file (or change it to reference
+``"django_mongodb_backend.fields.ObjectIdAutoField"``).
+
+Alternatively, you can use the following :djadmin:`startapp` template which
+includes this change:
+
+.. code-block:: bash
+
+    $ python manage.py startapp myapp --template https://github.com/mongodb-labs/django-mongodb-app/archive/refs/heads/5.1.x.zip
+
+(If you're using a version of Django other than 5.1.x, replace the two numbers
+to match the first two numbers from your version.)
+
+.. _configuring-databases-setting:
+
+Configuring the ``DATABASES`` setting
+=====================================
+
+After you've set up a project, configure Django's :setting:`DATABASES` setting
+similar to this::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django_mongodb_backend",
+            "HOST": "mongodb+srv://cluster0.example.mongodb.net",
+            "NAME": "my_database",
+            "USER": "my_user",
+            "PASSWORD": "my_password",
+            "PORT": 27017,
+            "OPTIONS": {
+                # Example:
+                "retryWrites": "true",
+                "w": "majority",
+                "tls": "false",
+            },
+        },
+    }
+
+For a localhost configuration, you can omit :setting:`HOST` or specify
+``"HOST": "localhost"``.
+
+:setting:`HOST` only needs a scheme prefix for SRV connections
+(``mongodb+srv://``). A ``mongodb://`` prefix is never required.
+
+:setting:`OPTIONS` is an optional dictionary of parameters that will be passed
+to :class:`~pymongo.mongo_client.MongoClient`.
+
+:setting:`USER`, :setting:`PASSWORD`, and :setting:`PORT` (if 27017) may also
+be optional.
```

Comment:
Can we have it say, if you do not require authentication, :setting:`USER` and :setting:`PASSWORD` should be omitted.

```suggestion
:setting:`USER` and :setting:`PASSWORD` should be omitted if you do not require authentication. 

:setting:`PORT` (if 27017) may also be optional.
```